### PR TITLE
added a page for my bot, u/profanitycounter

### PR DESCRIPTION
**Pull request by Aidgigi**

### CODE REVIEW

File: Bots/profanitycounter.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Reply to a post or comment with u/profanitcounter, and a modifier somewhere in your comment.
+Modifiers:
+  None: does nothing special, checks the profanity usage of the partent.
+  [self]: returns a profanity report of the user calling the bot.
+  [startstats]: registers the user in the stats system, and enables stat recording.
+  [stats]: summons the stat report of the user calling the bot
+
+#### Links
+Bot - [/u/DeepFryBot](https://www.reddit.com/user/profanitycounter)
```
Comment (eliassjogreen) [inline Bots/profanitycounter.md:21]:
```suggestion
Bot - [/u/profanitycounter](https://www.reddit.com/user/profanitycounter)
```

File: Bots/profanitycounter.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Creator - [/u/asdvek](https://www.reddit.com/user/Aidgigi)
```
Comment (eliassjogreen) [inline Bots/profanitycounter.md:23]:
```suggestion
Creator - [/u/Aidgigi](https://www.reddit.com/user/Aidgigi)
```

**Review by eliassjogreen — CHANGES_REQUESTED** [review]
Some links need to be changed i think

**Comment by Aidgigi** [comment]
yep, i was tired when i made that sorry heh

**Comment by eliassjogreen** [comment]
LGTM

**Pull request merged**